### PR TITLE
feat: petit bateau

### PR DESCRIPTION
**Pull request by charlesdassonville**

### CODE REVIEW

**Comment by Cj-Malone** [comment]
[CI output](https://data.alltheplaces.xyz/map.html?show=https://data.alltheplaces.xyz/ci/4363793413/petit_bateau/output.geojson) (which only runs for a couple minutes, hence the low POI count)

Could you either try and process it into a real url, or just throw it away. You should be able to override `pre_process_data` or `post_process_item`.

File: locations/spiders/petit_bateau.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import html
+
+from scrapy.spiders import SitemapSpider
+
+from locations.categories import Categories, apply_category
```
Comment (davidhicks) [inline locations/spiders/petit_bateau.py:5]: Unused imports from locations.categories

**Comment by charlesdassonville** [comment]
> [CI output](https://data.alltheplaces.xyz/map.html?show=https://data.alltheplaces.xyz/ci/4363793413/petit_bateau/output.geojson) (which only runs for a couple minutes, hence the low POI count)
> 
> Could you either try and process it into a real url, or just throw it away. You should be able to override `pre_process_data` or `post_process_item`.

i changed the url. Locally i got more than 3k pois

**Comment by Cj-Malone** [comment]
I meant to say `image` url in the output is bad, that should be cleaned or thrown away. Clearly my brain stopped working when I was writing the comment.

**Comment by Cj-Malone** [comment]
🎉 Congrats on the first merge and thank you 🎉

**Pull request merged**